modules/core/src/parallel.cpp

```diff
@@ -519,16 +539,22 @@ void parallel_for_(const cv::Range& range, const cv::ParallelLoopBody& body, dou
         }
     }
     else // nested parallel_for_() calls are not parallelized
-#endif // CV_PARALLEL_FRAMEWORK
     {
         CV_UNUSED(nstripes);
         body(range);
     }
 }
 
-#ifdef CV_PARALLEL_FRAMEWORK
+static
+void parallel_for_cb(int start, int end, void* data)
+{
+    const cv::ParallelLoopBody& body = *(const cv::ParallelLoopBody*)data;
```

Comment:
Maybe `CV_Assert(data)` (or DbgAssert) could be useful here to avoid static analyzers' warnings.